src/main/java/my/project/constants/Rule.java

```diff
@@ -0,0 +1,20 @@
+package my.project.constants;
+
+/**
+ * Created : 2020-11-23 오후 12:03
+ * Developer : Seo
+ */
+public class Rule {
+    public static final int ONE_GAME_PRICE = 1000;
+    public static final int MIN_NUMBER = 1;
+    public static final int MAX_NUMBER = 45;
+    public static final int ONE_GAME_COUNT = 6;
+
+    public static final int FIRST_PRICE = 2000000000;
+    public static final int THIRD_PRICE = 1500000;
+    public static final int FOURTH_PRICE = 50000;
+    public static final int FIFTH_PRICE = 5000;
```

Comment:
```suggestion
    public static final int FIRST_PRICE = 2_000_000_000;
    public static final int THIRD_PRICE = 1_500_000;
    public static final int FOURTH_PRICE = 50_000;
    public static final int FIFTH_PRICE = 5_000;
```
이렇게 표현하면 가독성에 더 좋답니다!